tests/testsuite/git_gc.rs

```diff
@@ -98,6 +98,7 @@ fn use_git_gc() {
 #[cargo_test]
 fn avoid_using_git() {
     if uses_gitoxide() {
+        // file protocol without git binary is currently not possible - needs built-in upload-pack
```

Comment:
Is there a gitoxide issue we can link to in this?

Reply:
A good idea, here is the fix: 3d0145effa57b78bf1390fe0468785d3a2d66df5.